app/config/sources.inc.php

```diff
@@ -654,6 +654,10 @@
     'snippets/2017/mar2017_b.lang' => [
         'deadline'          => '2017-02-24',
         'supported_locales' => ['id', 'zh-TW'],
+
+    'snippets/2017/apr2017.lang' => [
+        'deadline'          => '2017-03-21',
+        'supported_locales' => ['es-ES', 'de', 'fr', 'id', 'pt-BR', 'ru', 'zh-TW'],
```

Comment:
Remove the extra empty line before the code you're adding, and please fix the alphabetical order here ;-)